src/Solvers/krylov.jl

```diff
@@ -0,0 +1,87 @@
+import Krylov
+import Krylov.FloatOrComplex
+
+using KernelAbstractions: @index, @kernel
+
+function LinearAlgebra.norm(a::AbstractField; condition = nothing)
+    conditional_a = condition_operand(a, condition, 0)
+    result = zeros(a.grid, 1)
+    Base.mapreducedim!(x -> x * x, +, result, conditional_a)
+    return CUDA.@allowscalar sqrt(first(result))
+end
```

Comment:
I can't remember if I did this, but is there a reason to move this here? I think it better belongs in `Fields` module since it's not really specific to solvers or Krylov, for example.

Reply:
You implemented this function, but you overloaded `Statistics.norm`, whereas (to the best of my knowledge) `norm` should be overloaded from `LinearAlgebra`.

Reply:
I will move it back to `field.jl`.